pwd_guess_unit.py

```diff
@@ -2825,6 +2826,91 @@ def test_serialize(self):
         mock_serializer.serialize.assert_called_with('pass123', 0)
 
 
+class TestMainMultiGPU(unittest.TestCase):
+    def setUp(self):
+        self.test_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        shutil.rmtree(self.test_dir)
+
+    def get_available_gpus(self):
+        local_device_protos = device_lib.list_local_devices()
+        return [x.name for x in local_device_protos if x.device_type == 'GPU']
+
+    def _temp_path(self, name):
+        return os.path.join(self.test_dir, name)
+
+    @unittest.skipIf(len(get_available_gpus(None)) < 2,
```

Comment:
It would be better to make get_available_gpus an @staticmethod to avoid passing in None as the self parameter.